google-cloud-bigquery/lib/google/cloud/bigquery/table.rb

```diff
@@ -1217,6 +1227,13 @@ def to_gapi
             @gapi
           end
 
+          def time_partitioning type: "DAY", expiration_ms: nil
```

Comment:
I believe this should be a write/mutator method: `time_partitioning=`. Also, it should accept the same type that is returned by `time_partitioning`, above.